test/tests-levenshtein.cpp

```diff
@@ -4,7 +4,7 @@
 #include <string_view>
 #include <vector>
 
-#include "../src/levenshtein.hpp"
+#include <rapidfuzz/levenshtein.hpp>
```

Comment:
This should be changed in the other tests aswell

Reply:
done in c1ef31ba & 75dcc46b